### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by SimonFair** [virtio-win/kvm-guest-drivers-windows#838] [General]
How often do you release new tags and iso packages?

**Comment by vrozenfe** [comment]
Approximately every three months. 
Upstream "stable" rpm and iso releases usually aligned with our internal releases,
which happen every three months.

**Comment by GDeCecco-CenterGrid** [comment]
Is there a page that lists all the releases, their beta/stable status, and release notes with known issues?  I've looked but cannot find such a page.  Your drivers are used in the enterprise space and such a status page would be great for our internal processes, internal documentation, and ticketing system.
